docs/CONTRIBUTING.md

```diff
@@ -148,7 +148,8 @@ $ make e2e-docker-push
 After Docker images are pushed to the inner Docker registry, run e2e tests:
 
 ```sh
-$ kubectl apply -f tests/manifests/e2e/e2e.yaml
+# replace 127.0.0.1:5000 with localhost:5000 to make it work with containerd
+$ set 's/127.0.0.1:5000/localhost:5000/g' tests/manifests/e2e/e2e.yaml | kubectl apply -f -
```

Comment:
kind is planning to support insecure registries in the future. for now, we need to use `localhost:5000`.

Reply:
Maybe we should replace `127.0.0.1` to `localhost` in our CI scripts instead of here.

Reply:
you mean `tests/manifests/e2e/e2e.yaml`? that would be better
I am going to update it to make it work both in local kind and IDC e2e environments.

Reply:
This doesn't work as expected:
```
cjc:tidb-operator changjunchang$ set 's/127.0.0.1:5000/localhost:5000/g' tests/manifests/e2e/e2e.yaml | kubectl apply -f -
error: no objects passed to apply
```

Reply:
oh, I made a typo here, `set` should be `sed`
however, I think it's better to update e2e.yaml directly. I think it will not break e2e job.

Reply:
what about change `127.0.0.1` to `localhost`directly

Reply:
yes, updated in https://github.com/pingcap/tidb-operator/pull/890/commits/28e8f03c29958c970316de65569e1a12ed9ddb4c